fraqbot/Local/shakespeare.py

```diff
@@ -0,0 +1,49 @@
+from Legobot.Lego import Lego
+import logging
+import os
+import random
+import sys
+
+logger = logging.getLogger(__name__)
+
+insult_array = []
+
+LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
+if LOCAL_DIR not in sys.path:
+    sys.path.append(LOCAL_DIR)
+
+with open(os.path.join(LOCAL_DIR, 'lists/quotes.txt')) as my_file:
```

Comment:
This should probably be part of the init of the Shakespeare class instead of loaded outside of it, which means it becomes a property of the class -- self.insult_array